libs/platform/src/lib/components/dynamic-page/dynamic-page.component.ts

```diff
@@ -239,15 +239,17 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
      * set position for tabs and tabbed content's position relative to the tabs on scrolling
      */
     private _setTabsPosition(): void {
-        const tabList: HTMLElement = this._elementRef.nativeElement.querySelector('.fd-tabs');
-        if (tabList) {
-            tabList.style.top = this.header.nativeElement.offsetHeight + 'px';
-            tabList.style.position = 'fixed';
-            tabList.style.left = '0';
-            tabList.style.right = '0';
-            this.tabs.forEach((element) => {
-                element.contentTop = tabList.offsetHeight + 'px';
-            });
+        if (this.contentContainer) {
+            const tabList: HTMLElement = this.contentContainer.nativeElement.querySelector('.fd-tabs');
+            if (tabList) {
+                tabList.style.top = this.header.nativeElement.offsetHeight + 'px';
+                tabList.style.position = 'fixed';
+                tabList.style.left = '0';
+                tabList.style.right = '0';
+                this.tabs.forEach((element) => {
+                    element.contentTop = tabList.offsetHeight + 'px';
+                });
+            }
         }
```

Comment:
decrease nesting

```suggestion
        if (!this.contentContainer) {
            return;
        }

        const tabList: HTMLElement = this.contentContainer.nativeElement.querySelector('.fd-tabs');
        if (!tabList) {
            return;
        }
        
        tabList.style.top = this.header.nativeElement.offsetHeight + 'px';
        tabList.style.position = 'fixed';
        tabList.style.left = '0';
        tabList.style.right = '0';
        this.tabs.forEach((element) => {
            element.contentTop = tabList.offsetHeight + 'px';
        });
```

Reply:
fixed as suggested, thanks for pointing this out!